roles/gpu_operator_deploy_from_operatorhub/tasks/deploy_clusterpolicy.yml

```diff
@@ -48,26 +48,12 @@
 - name: Create the clusterPolicy CR for the GPU Operator
   command: oc apply -f "{{ artifact_extra_logs_dir }}/gpu_operator_clusterpolicy.json"
 
-- name: Test if the ClusterPolicy has the 'validator' stanza
-  command: oc get clusterpolicies.nvidia.com -ojsonpath='{range .items[*]}{.spec.validator.version}{end}'
-  register: gpu_operator_atleast_v170
-  failed_when: false
+- name: Get the GPU Operation version
+  include_role:
+    name: gpu_operator_get_csv_version
 
 - name: Create the namespace for the GPU Operator (only < v1.7.0)
-  when: not gpu_operator_atleast_v170.stdout
+  when: gpu_operator_version < "1.7.0"
```

Comment:
Does string comparison work like this? Is it alphabetical?

Reply:
Wouldn't it break with 2-digit version components? e.g. 1.10.2

Reply:
yes for the part "works like this", I double checked,
but indeed it will break > 1.9 ;/

Reply:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

Reply:
great, thanks, I'll update the PR

Reply:
fixed